incubator/hnc/pkg/reconcilers/hnc_config.go

```diff
@@ -133,17 +157,25 @@ func (r *ConfigReconciler) createObjectReconcilers(inst *api.HNCConfiguration) e
 	// using the latest forest structure.
 	r.Forest.Lock()
 	defer r.Forest.Unlock()
+
+	hasError := false
 	for _, t := range inst.Spec.Types {
 		gvk := schema.FromAPIVersionAndKind(t.APIVersion, t.Kind)
 		if r.Forest.HasTypeSyncer(gvk) {
 			continue
 		}
 		if err := r.createObjectReconciler(gvk); err != nil {
-			return fmt.Errorf("Error while trying to create ObjectReconciler: %s", err)
+			msg := fmt.Sprintf("Error while trying to create ObjectReconciler for type %s: %s", gvk, err)
```

Comment:
"Error while" makes sense for a log message, but not really for a status. The status should be more something like "Couldn't create reconciler for type %s: %s".

Also, if you want to log this here, log it as a real error: `r.log.Error(err, "Couldn't create reconciler", "type", gvk)`

Reply:
Thanks for the suggestion! Moved the error handling in `createObjectReconciler`. Logged it as real error there and also change the status message as you suggested. PTAL.